.github/workflows/enable-pack-build-and-test.yaml

```diff
@@ -61,60 +60,35 @@ jobs:
       # TODO: Make sure this actually constitutes "repository activity"
       - name: Conditionally set this Workflow to active (attempt to prevent 60d disabling of itself)
         shell: bash
+        env:
+          WORKFLOW_PATH: repos/${{ github.repository }}/actions/workflows/enable-pack-build-and-test.yaml
         run: |
           # NOTE: "gh workflow list" does not currently support json output
           # Relates to: https://github.com/cli/cli/issues/5025
-          if [[ $(gh api --method GET "repos/${{ github.repository }}/actions/workflows/enable-pack-build-and-test.yaml" --jq .state) != "active" ]]; then
-            gh workflow enable "${{ github.workflow }}" --repo "${{ github.repository }}"
+          if [[ $(gh api --method GET "${WORKFLOW_PATH}" --jq .state) != "active" ]]; then
+            gh api --method PUT "${WORKFLOW_PATH}/enable"
```

Comment:
Since we're using `gh api` here, we don't actually have to check to see if `!= "active"`, because the API does not care what the current state is, it just enables it. The restriction was in the `gh workflow` tool.

So, wdyt? Drop the condition to reduce the number of API calls?

Reply:
> Drop the condition to reduce the number of API calls?

Yeah, might as well.